### PR TITLE
Handle invalid URL failures

### PR DESCRIPTION
**Pull request by samuelli**

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -35,18 +35,32 @@ app.get('/', (request, response) => {
...
-  const result = await renderer.serialize(request.params.url, request.query, config).catch((err) => console.error(err));
-  response.status(result.status).send(result.body);
+  try {
+    const result = await renderer.serialize(request.params.url, request.query, config);
+    response.status(result.status).send(result.body);
+  } catch (err) {
+    let message = `Cannot render ${request.params.url}`;
+    if (err && err.message)
+      message += ` - "${err.message}"`;
+    response.status(400).send(message);
```
Comment (aomarks) [inline src/main.js]: 400 or 500 (same question for below)?

File: src/main.js [hunk 2]
```diff
@@ -80,18 +94,22 @@ const appPromise = chromeLauncher.launch({
 
 
 let exceptionCount = 0;
-function logUncaughtError(error) {
+async function logUncaughtError(error) {
   console.error('Uncaught exception');
   console.error(error);
   exceptionCount++;
   // Restart instance due to lots of failures.
```
Comment (aomarks) [inline src/main.js]: Update comment (no longer lots of failures).

File: src/main.js (hunk 1 above)
Comment (samuelli) [inline src/main.js]: Unclear. But for the tested cases, eg. invalid URLs, 400 is more appropriate. The client requested a bad URL, it's not really an error on the server side.

File: src/main.js (hunk 2 above)
Comment (samuelli) [inline src/main.js]: Oops this shouldn't be 0, my bad.

**Pull request merged**